plugins/org.jkiss.dbeaver.ext.mssql/src/org/jkiss/dbeaver/ext/mssql/edit/SQLServerViewManager.java

```diff
@@ -102,12 +102,15 @@ private void createOrReplaceViewQuery(DBRProgressMonitor monitor, DBCExecutionCo
             actions.add(new SQLDatabasePersistAction("Set current database", "USE " + DBUtils.getQuotedIdentifier(procDatabase), false)); //$NON-NLS-2$
         }
 
+        String viewDDL = view.getDataSource().isAtLeastV16()
+                         ? view.getDDL().replace("CREATE", "CREATE OR ALTER")
```

Comment:
This will replace ALL mentions of the word `CREATE`. We need a more sophisticated solution.

Reply:
@ShadelessFox  Yes, tested it, accidentally didn't mark as a draft mr. Fixed.